pkg/manager/member/startscript/v2/pd_start_script.go

```diff
@@ -77,13 +80,14 @@ const (
 	pdStartScript = `
 PD_POD_NAME=${POD_NAME:-$HOSTNAME}
 PD_DOMAIN={{ .PDDomain }}
+threshold={{ .PDStartTimeout }} 
 
 elapseTime=0
 period=1
-threshold=30
 while true; do
     sleep ${period}
     elapseTime=$(( elapseTime+period ))
+	echo "waiting threshold for pd ${threshold}" 
```

Comment:
update the content of this script (in a ConfigMap) will cause existing PD rolling to restart. How about just replacing `threshold=30` with `threshold={{ .PDStartTimeout }}`?

Reply:
updated. can you pls help review again? thx